docs/guide/application/commands.md

```diff
@@ -229,15 +229,15 @@ final class EventManagementApplication implements EventManagementApplicationInte
         );
 
         $middleware->bind(
-            LogMessageDispatch::class,
-            fn () => new LogMessageDispatch(
+            LogBusDispatch::class,
```

Comment:
Have accidentally renamed this i.e. PHPStorm has done it without me realising! Need to fix as I think this is an unnecessary change.